core/json/tx.go

```diff
@@ -102,7 +102,7 @@ func (v *Vin) MarshalJSON() ([]byte, error) {
 // getrawtransaction and decoderawtransaction use the same structure.
 type Vout struct {
 	Coin         string             `json:"coin,omitempty"`
-	CoinId       uint16             `json:"coinid,omitempty"`
```

Comment:
why not only remove the omitempty of tag ?

Reply:
In order not to affect the previous lock vout

```json
{
            "scriptPubKey": {
              "asm": "OP_RETURN OP_MEER_LOCK 007841cb02",
              "hex": "6ac305007841cb02",
              "type": "LockAmount"
            }
}
```
if remove omitempty, the lock vout will be this

```json
{
            "coinid": 0,
            "scriptPubKey": {
              "asm": "OP_RETURN OP_MEER_LOCK 007841cb02",
              "hex": "6ac305007841cb02",
              "type": "LockAmount"
            }
}
```

Reply:
removing the `omitempty` is enough for us